SU-SWS/jumpstart_ui
# D8CORE-106: Media with caption paragraph

# READY FOR REVIEW

# Summary
- Gives the right aligned treatment to the caption same same as hero.

# Review By (Date)
- Jan 29th.

# Urgency
- Low

# Steps to Test

1. See tugboat build at: https://github.com/SU-SWS/acsf-cardinalsites/pull/51
2. Log in and create a new stanford_page
3. Create a new Media with caption paragraph
4. Fill in fields with an image and lots of text
5. Save new node and review front end
6. Review code

# Affected Projects or Products
- D8CORE
- Stanford Media Component.

# Associated Issues and/or People
- https://github.com/SU-SWS/acsf-cardinalsites/pull/51
- D8CORE-106

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
